plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqCodeTest/utils/UTGChatUtil.kt

```diff
@@ -109,10 +130,14 @@ fun runBuildOrTestCommand(
     }
 
     val processBuilder = ProcessBuilder()
-        .command(listOf(command) + args)
-        .directory(File(repositoryPath))
+        .command(listOf("zsh", "-c", "source ~/.zshrc && $command ${args.joinToString(" ")}"))
+        .directory(packageRoot)
         .redirectErrorStream(true)
 
+    val env = processBuilder.environment()
+
+    env["PATH"] = System.getenv("PATH")
```

Comment:
processes should be started using GeneralCommandLine. see its doc string for an explanation on how the environment variable logic works